File: add/codesnippet/CPP/p-system.diagnostics.fil_19_1.cpp
private:
    void GetFileName()
    {
        // Get the file version for the notepad.
        FileVersionInfo^ myFileVersionInfo =
            FileVersionInfo::GetVersionInfo( Environment::SystemDirectory + "\\Notepad.exe" );

        // Print the file name.
        textBox1->Text = String::Concat( "File name: ", myFileVersionInfo->FileName );
    }

File: add/codesnippet/CSharp/p-system.diagnostics.fil_19_1.cs
    private void GetFileName()
    {
        // Get the file version for the notepad.
        FileVersionInfo myFileVersionInfo =
            FileVersionInfo.GetVersionInfo(Environment.SystemDirectory + "\\Notepad.exe");

        // Print the file name.
        textBox1.Text = "File name: " + myFileVersionInfo.FileName;
    }